### DISCUSSION TITLE
[manager] "instance deployment information exception" verify cluster failed

### DISCUSSION THREAD

**Discussion by KeithTt** [apache/doris#14274] [A - General / Q&A]
I create a managed cluster on doris manager, but verify failed.

![图片](https://user-images.githubusercontent.com/15189954/201807703-c8391bfd-2194-4110-80fc-2b9f94565889.png)

Then I check the logs, it can not find the pid path, and I create soft link from working dir to `fe` and `be`, and `fe` verify success, but `be` still failed, there are no exceptions in the logs...

![图片](https://user-images.githubusercontent.com/15189954/201822156-72dfe475-2cdc-49d1-a647-e7c4ffbfd70f.png)

What have I ignored?

**Comment by yxy0502** [comment]
BE监测心跳BE_PID_NAME为palo_be，实际启动为doris_be。修改所有BE启动脚本start_be.sh最后
if [ ${RUN_DAEMON} -eq 1 ]; then
    nohup $LIMIT ${DORIS_HOME}/lib/doris_be "$@" >> $LOG_DIR/be.out 2>&1 < /dev/null &
else
    export DORIS_LOG_TO_STDERR=1
    $LIMIT ${DORIS_HOME}/lib/doris_be "$@" 2>&1 < /dev/null

doris_be ---> palo_be
